### ISSUE TITLE
Source code

### ISSUE THREAD

**Issue by LifeIsAParadox** [slushiegoose/Town-Of-Us#21]
do you plan to release the source code?

**Comment by gendelo3** [comment]
you should check the releases, the source code is available as zip file.

https://github.com/slushiegoose/Town-Of-Us/releases

**Comment by realDripDrip** [comment]
No, it isnt. You can only download the files containing in the master wich do not contain the source.

**Comment by manniL** [comment]
Lets merge this issue and #21

**Comment by soerface** [comment]
@manniL you mean #16 :)

**Comment by manniL** [comment]
whoops, yup.
